src/core_plugins/input_control_vis/public/components/editor/control_editor.js

```diff
@@ -0,0 +1,111 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+export class ControlEditor extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isEditorVisible: true
+    };
+  }
+
+  handleToggleControlVisibility() {
+    this.setState({
+      isEditorVisible: !this.state.isEditorVisible
+    });
+  }
+
+  render() {
+    let editor;
+    if (this.state.isEditorVisible) {
+      editor = (
+        <div>
+          <div className="kuiSideBarFormRow">
+            <label className="kuiSideBarFormRow__label">
+              Label
+            </label>
+            <div className="kuiSideBarFormRow__control kuiFieldGroupSection--wide">
+              <input
+                className="kuiTextInput"
+                type="text"
+                value={this.props.controlParams.label}
```

Comment:
Are you sure about that? From working in x-pack-kibana I know that from 6.1 onward there is an eslint rule enforcing no spaces inside of `{}`. 
https://github.com/elastic/kibana/blob/master/packages/eslint-config-kibana/.eslintrc.js#L89-L100

Reply:
Whoa you are right (https://github.com/elastic/x-pack-kibana/pull/2371 and https://github.com/elastic/kibana/pull/13323 did all the conversions), sorry for the misdirection @nreese and thanks for pointing that out @archanid.  Looks like we switched it over to match the AirBnb style guide. I guess the react style guide should be updated because all of the examples in there use the old style of spaces inside.

Reply:
I filed https://github.com/elastic/kibana/issues/13846 to keep track.